core/src/com/unciv/models/ruleset/Victory.kt

```diff
@@ -170,6 +181,19 @@ class Milestone(val uniqueDescription: String, private val parentVictory: Victor
                 else
                     "{$uniqueDescription} (${amountDone.tr()}/${amountToDo.tr()})"
             }
+            MilestoneType.MoreCountableThanEachPlayer -> {
+                val amountDone = civInfo.gameInfo.civilizations.count {
```

Comment:
I would separate this to
val relevantCivs = civInfo.gameInfo.civilizations.count {
                    it != civInfo && it.isMajorCiv() && it.isAlive()
                }
val amountDone = relevantCivs.count{ countables ... }
val amountToDo = relevantCivs.count()